blog/2021-05-15-announcing-stryker-5-brave-new-dimension.md

```diff
@@ -0,0 +1,141 @@
+
+If you're already using Stryker, you can install this latest version with your package manager of choice.
+
+```shell
+npm install --save-dev @stryker-mutator/core
+```
+
+Don't forget to do the same for any plugins you might be using. For example,
+
+```shell
+npm install --save-dev @stryker-mutator/mocha-runner
+```
+
+With that out of the way, let's dive into the new stuff!
+
+## 👪 Name change
+
+That's right. We've rebranded 'Stryker for JavaScript and friends' as simply **StrykerJS**. This change makes it easier to distinguish StrykerJS from other frameworks in the Stryker family (Stryker.NET and Stryker4s). It also allows for better use of Twitter real estate 😊.
+
+Don't worry; the names of the NPM packages haven't changed. So it is purely a branding thing.
+
+## ✨ New dimension
+
+The mutation test report has always been the bread and butter of Stryker. Either in your browser or console, it is where you find out where the killed and survived mutants are. But the next question you might ask is: which tests were responsible? Stryker knows but wasn't telling you. Until now!
+
+Behold, the new dimension you'll find in your reports—the test view 🧪.
+
+![](/images/blogs/test-view-html-1.png)
+![](/images/blogs/test-view-html-2.png)
+
+Or using the `clear-text` reporter:
+
+![](/images/blogs/test-view-console.png)
+
+The test view tells you at a glance which tests are _killing_ mutants, which are merely _covering_ mutants (without killing) and which are _not covering_ any mutants. Furthermore, you can determine which mutants were covered or killed by that particular test case. Finding out so much information about your tests has never been this easy!
+
+A couple of caveats to point out:
+
+- The `Covering` state will only be available when selecting the [`perTest` coverage analysis setting](https://stryker-mutator.io/docs/stryker-js/configuration#coverageanalysis-string).
+- StrykerJS always runs your tests with `bail` mode active (or `failFast` in jasmine), making your test runner stop after the first failing test. As a result, a mutant can only be killed by one test. If you have overlap in asserts across tests, other tests might be marked as "Covering", while they would kill a mutant when run in isolation.
+- Some mutants are executed as soon as your file _is loaded_ instead of during _test execution_. For example:
+  ```diff
+  -export const hello ='👋'
+  +export const hello = ''
+  ```
+  Tests are never able to cover such mutants (yet they can still kill them).
```

Comment:
This confused me. A test never covers a mutant but can still kill it? What is this magic

Reply:
How would you describe it?

Reply:
I'm not sure what you mean by it never covering such mutants but still being able to kill them, so I don't know how I'd describe it 😅. Are you talking about static mutants?